packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -26,6 +26,60 @@ import {
 import FormSubHeader from '../form-sub-header';
 import { splitValidationResultTypes } from '../real-account-signup/helpers/utils';
 
+export interface TFinancialDetails {
```

Comment:
there are many common types between `TFinancialDetails` and `TFinancialInformationAndTradingExperience`. can we have another type for them and use that instead of repeating them?

Reply:
since some of the typescript item is required, I didn't combine them into 1 type but i have remove the duplicated items.